modules/performance/extension/siddhi-execution-performance/component/src/main/java/org/wso2/extension/siddhi/execution/throughput/CalculatePerformanceStreamProcessorExtension.java

```diff
@@ -117,6 +117,7 @@
     private String executionType;
     private ExecutorService executorService;
     private boolean flag;
+    String siddhiAppContextName = "";
```

Comment:
No need to initialize the variable here. Because you will be assigning a value to the variable before you use it in your code.